internal/domain/client.go

```diff
@@ -57,3 +68,54 @@ const (
 	DownloadClientTypeWhisparr     DownloadClientType = "WHISPARR"
 	DownloadClientTypeReadarr      DownloadClientType = "READARR"
 )
+
+func (c DownloadClient) BuildLegacyHost() string {
+	if c.Type == DownloadClientTypeQbittorrent {
+		return c.qbitBuildLegacyHost()
+	}
+	return ""
+}
+
+func (c DownloadClient) qbitBuildLegacyHost() string {
+	// parse url
+	u, _ := url.Parse(c.Host)
+
+	// reset Opaque
+	u.Opaque = ""
+
+	// set scheme
+	scheme := "http"
+	if u.Scheme == "http" || u.Scheme == "https" {
```

Comment:
bad branch

Reply:
It's apparent how it would be that. This is here for legacy reasons when we constructed the `host` for qbit where it was valid to not have the scheme in the host field.

Reply:
understood, but the else branch is the exact same code... so this doesn't need to be in a conditional statement (unless if that's the bug here 😉)

Reply:
Sometimes it's just right in front of you. HA! Will resolve.